subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeMultipleCppProjectIntegrationTest.groovy

```diff
@@ -17,17 +17,146 @@
 package org.gradle.ide.xcode
 
 import org.gradle.ide.xcode.fixtures.AbstractXcodeIntegrationSpec
+import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import org.gradle.nativeplatform.fixtures.app.CppAppWithLibraries
 import org.gradle.nativeplatform.fixtures.app.CppAppWithLibrary
+import org.gradle.util.Requires
+import org.gradle.util.TestPrecondition
+import spock.lang.IgnoreIf
 
 import static org.gradle.ide.xcode.internal.XcodeUtils.toSpaceSeparatedList
 
+@Requires(TestPrecondition.XCODE)
+@IgnoreIf({GradleContextualExecuter.embedded})
 class XcodeMultipleCppProjectIntegrationTest extends AbstractXcodeIntegrationSpec {
     def setup() {
         settingsFile << """
             include 'app', 'greeter'
         """
     }
 
```

Comment:
I think we also need to have a couple of tests where the projects fail to build so we can confirm:

- Our test fixture fails when we expect it to
- xcodebuild reports the failure from our Gradle builds
- xcodebuild shows reasonable output from Gradle builds

Reply:
Does [this test](https://github.com/gradle/gradle/pull/2923/commits/94464234ffa7c617ac830982e28ff230f57f4422) make sense?